.vscode/tasks.json

```diff
@@ -1,30 +1,37 @@
-// See https://go.microsoft.com/fwlink/?LinkId=733558
-// for the documentation about the tasks.json format
 {
-	"version": "2.0.0",
-	"tasks": [
-		{
-			"label": "watch",
-			"type": "shell",
-			"command": "yarn watch",
-			"problemMatcher": "$tsc-watch",
-			"isBackground": true,
-			"presentation": {
-				"reveal": "never"
-			},
-			"runOptions": {
-				"runOn": "folderOpen"
-			},
-			
-		},
-		{
-			"label": "compile",
-			"type": "shell",
-			"command": "yarn compile",
-			"group": {
-				"kind": "build",
-				"isDefault": true
-			}
-		}
-	]
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "label": "install",
+      "type": "shell",
+      "command": "yarn install",
+      "problemMatcher": [],
+      "presentation": {
+        "reveal": "silent"
+      }
+    },
```

Comment:
This will run a install script every time I open vscode because the watch script depends on it. I really don't want this. Is better the programmer just manually run install the first time he open the project.